### PR TITLE
Added a note about where to find the container source.

### PR DESCRIPTION
**Pull request by code128**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,8 +201,11 @@ <h3>What's next?</h3>
 				<a href="https://console.cloud.google.com/run/detail/{{.Region}}/{{.Service}}/metrics?project={{.Project}}" target="_blank" rel="noopener">
 					VIEW IN CLOUD CONSOLE</a>
 				{{ end }}
-			</p>
+			</p>            
 		</div>
+        <p class="footnote">Find the source code for this container at 
+             <a href="https://github.com/GoogleCloudPlatform/cloud-run-hello">Github.</a>
```
Comment (ahmetb) [inline index.html:207]: this container → this application
at Github → on GitHub
?

File: index.html [hunk 2]
```diff
@@ -197,8 +201,11 @@ <h3>What's next?</h3>
 				<a href="https://console.cloud.google.com/run/detail/{{.Region}}/{{.Service}}/metrics?project={{.Project}}" target="_blank" rel="noopener">
 					VIEW IN CLOUD CONSOLE</a>
 				{{ end }}
-			</p>
+			</p>
```
Comment (ahmetb) [inline index.html:204]: trailing whitespace here and on line 75

File: index.html (hunk 1 above)
Comment (ahmetb) [reply index.html:207]: also this is mixing tabs and spaces for indentation fwiw 😇

**Comment by steren** [comment]
I'm actually not sure we want users who deploy to go to GitHub. I prefer to keep the links to the Cloud Run docs only

**Pull request closed without merging**